### PR TITLE
Feature/SOF-7287 feature: add JL drawer to MD

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: src/components/drawer_session/JupyterLiteSessionDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    messageHandler = new MessageHandler();
+
+    // eslint-disable-next-line no-useless-constructor
+    constructor(props: JupyterLiteTransformationProps) {
+        super(props);
+    }
+
+    componentDidMount() {
+        this.messageHandler.addHandlers("set-data", [this.handleSetMaterials]);
+        this.messageHandler.addHandlers("get-data", [this.returnSelectedMaterials]);
```
Comment (timurbazhirov) [inline src/components/drawer_session/JupyterLiteSessionDrawer.tsx]: Seems like code duplicate Let's reuse this through a parent

File: dist/tests-legacy/cucumber/support/widgets/alert/s_alert_widget.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      if (this.isVisible()) {
+        _logger.logger.debug("sAlertSuccess closed");
+        this.close();
+        // eslint-disable-next-line no-throw-literal
+        throw "Something is wrong: there should be no sAlertSuccess shown!";
+      }
+    });
+  }
+}
+exports.SAlertWidget = SAlertWidget;
```
Comment (timurbazhirov) [inline dist/tests-legacy/cucumber/support/widgets/alert/s_alert_widget.js]: Wtf are all these extra files???

Comment (VsevolodX) [reply dist/tests-legacy/cucumber/support/widgets/alert/s_alert_widget.js]: They came from commiting dist

File: src/components/drawer_session/JupyterLiteSessionDrawer.tsx (hunk 1 above)
Comment (VsevolodX) [reply src/components/drawer_session/JupyterLiteSessionDrawer.tsx]: Added base and inheritance

File: src/components/drawer_session/JupyterLiteSessionDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class JupyterLiteSessionDrawer extends BaseJupyterLiteSessionComponent {
+    render() {
+        const { show, onHide, containerRef } = this.props;
+
+        return (
+            <div style={{ display: show ? "block" : "none" }}>
+                <ResizableDrawer open={show} onClose={onHide} containerRef={containerRef}>
+                    <JupyterLiteSession
+                        originURL="https://jupyterlite.mat3ra.com"
```
Comment (timurbazhirov) [inline src/components/drawer_session/JupyterLiteSessionDrawer.tsx]: Remove originURL

defaultNotebook should be set in Base

messageHandler should be created inside Base

File: src/components/3d_editor/advanced_geometry/python_transformation/JupyterLiteTransformation.tsx [hunk 4]
```diff
@@ -162,7 +83,7 @@ class JupyterLiteTransformation extends React.Component<
                             }}
                         >
                             <JupyterLiteSession
-                                defaultNotebookPath={DEFAULT_NOTEBOOK_PATH}
+                                defaultNotebookPath={this.DEFAULT_NOTEBOOK_PATH}
```
Comment (timurbazhirov) [inline src/components/3d_editor/advanced_geometry/python_transformation/JupyterLiteTransformation.tsx]: Should be set in Base

**Review by timurbazhirov — APPROVED** [review]

File: src/components/3d_editor/advanced_geometry/python_transformation/JupyterLiteTransformation.tsx (hunk 4 above)
Comment (VsevolodX) [reply src/components/3d_editor/advanced_geometry/python_transformation/JupyterLiteTransformation.tsx]: Finally moved!

**Pull request merged**